src/pages/_middleware.ts

```diff
@@ -2,6 +2,9 @@ import { NextResponse } from 'next/server'
 import type { NextFetchEvent, NextRequest } from 'next/server'
 import redirects from 'data/_redirects.generated.json'
 
+export type OptInPlatformOption = 'vault-io' | 'waypoint-io' | 'learn'
+const OPT_IN_MAX_AGE = 60 * 60 * 24 * 90 //90 days
```

Comment:
Anyone have suggestions on this lenght?

Reply:
That makes sense! It just sounded v long when I was implementing but I'll update. I think you're right, might as well assume folks can opt in until GA.